### ISSUE TITLE
Accessibility violations in IdeNavigation

### ISSUE THREAD

**Issue by marekdano** [carbon-design-system/ibm-cloud-cognitive#322]
## Detailed description

> Describe in detail the issue you're having.

Accessibility violation 
`Text contrast of 1.16 with its background is less than the WCAG AA minimum requirements for text of size 14px and weight of 400` 

> Is this issue related to a specific component?

IdeNavigation

> What did you expect to happen? What happened instead? What would you like to
> see changed?

I'd expect no violations for icon buttons in the footer of the component

> What browser are you working in?

Chrome with [A11y checker
](https://chrome.google.com/webstore/detail/ibm-equal-access-accessib/lkcagbfjnkomcinoddgooolagloogehp)
> What version of the Carbon Design System are you using?

> What offering/product do you work on? Any pressing ship or release dates we
> should be aware of?

event-streams on IBM cloud 

<img width="1739" alt="Screenshot 2021-02-08 at 16 40 22" src="https://user-images.githubusercontent.com/1914165/107252571-64136f80-6a2d-11eb-9315-c00eb914d61d.png">

**Comment by andrea-gm** [comment]
@asfordmatt @marekdano was this resolved? can we close this issue?

**Comment by marekdano** [comment]
Sorry, @andrea-gm I can still see the issue [here](https://ibm-cloud-cognitive.netlify.app/?path=/story/legacy-cd-ai-idenavigation-ide-navigation-component-set--all-components-used-together) :( 

<img width="1741" alt="Screenshot 2021-04-27 at 10 37 00" src="https://user-images.githubusercontent.com/1914165/116220768-e23a0500-a744-11eb-8b92-41d6eb5dc57e.png">
